### PR TITLE
Fix cloudex configuration

### PR DESCRIPTION
**Pull request by joshsmith**
# What's in this PR?

This fixes some Cloudex config issues that made it into a merge.

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -62,11 +62,10 @@ config :code_corps, CodeCorps.Mailer,
 config :code_corps,
   postmark_receipt_template: "123"
 
-# If the dev environment has no CLOUDEX_API_KEY set, we want the app to still run,
-# with cloudex in test API mode
+# If the dev environment has no CLOUDEX_API_KEY set, we want the app
+# to still run, with cloudex in test API mode
 if System.get_env("CLOUDEX_API_KEY") == nil do
-  config :cloudex, :cloudinary_api, Cloudex.CloudinaryApi.Test
```
Comment (begedin) [inline config/dev.exs:68]: This is actually a documented `Cloudex` approach of enabling test mode for the API. Don't have a link right now, but should probably look into the Clodex docs to see what it does. Not sure if we wanna remove it.

File: config/dev.exs [hunk 2]
```diff
@@ -62,11 +62,10 @@ config :code_corps, CodeCorps.Mailer,
...
-  config :cloudex, api_key: "test_key", secret: "test_secret", cloud_name: "test_cloud_name"
-
   IO.puts("NOTE: No Cloudex configuration found. Cloudex is runnning in test mode.")
+  config :code_corps, :cloudex, CloudexTest
```
Comment (begedin) [inline config/dev.exs:69]: I think this is what fixes it. The removal of the other change probably doesn't break anything right now, but might be misleading.

File: lib/code_corps/helpers/cloudex_test.ex [hunk 3]
```diff
@@ -1,4 +1,4 @@
-defmodule Cloudex.CloudinaryApi.Test do
```
Comment (begedin) [inline lib/code_corps/helpers/cloudex_test.ex:1]: This is probably the other part of the fix.

**Review by begedin — COMMENTED** [review]
Looking from my phone, it's hard to judge, but you might be tackling two separate things here. Look into the cloudex docs to see how they handle testing and see which parts we need to change here and which we don't.

It's possible this is all good, but I'm not sure from my phone.

File: config/dev.exs (hunk 1 above)
Comment (joshsmith) [reply config/dev.exs:68]: We don't make use of the API, so don't really need it right now.

File: config/dev.exs (hunk 2 above)
Comment (joshsmith) [reply config/dev.exs:69]: Yeah, I'm going to move this out of `helpers` too and into a dedicated `cloudex`.

**Review by begedin — APPROVED** [review]
Considering your comments, looks good to go.

**Pull request merged**